pytorch_lightning/trainer/trainer.py

```diff
@@ -1733,7 +1735,7 @@ def checkpoint_callbacks(self) -> List[ModelCheckpoint]:
         return [c for c in self.callbacks if isinstance(c, ModelCheckpoint)]
 
     @property
-    def resume_from_checkpoint(self) -> Optional[Union[str, Path]]:
+    def resume_checkpoint_path(self) -> Optional[Union[str, Path]]:
```

Comment:
- let's use the typehint _PATH here too? 
- should this property be deprecated as well given it's entirely user input?

Reply:
yes on second thought i think this can be deprecated entirely. user passes`ckpt_path`, `resume_start`/`resume_end` will be called to set and forget this in checkpoint_connector